invenio_app_ils/records/api.py

```diff
@@ -92,7 +94,7 @@ def validate(self, **kwargs):
             raise IlsValidationError(
                 description="Record validation error",
                 errors=errors,
-                original_exception=jve
+                original_exception=jve,
```

Comment:
is `jve` `JsonValidationError` or something else cryptic?

Reply:
it is just above:
`except ValidationError as jve:`